langlib/lang.runtime/src/main/ballerina/runtime.bal

```diff
@@ -75,6 +100,10 @@ public type StackFrame readonly & object {
 
 # Returns a stack trace for the current call stack.
 #
+# ```ballerina
+# runtime:getStackTrace();
```

Comment:
Why aren't we doing the assignment here?

Reply:
```suggestion
# runtime:StackFrame[] stackTrace = runtime:getStackTrace();
```

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-lang/pull/39006/commits/8348db48185e6e41ea644ddbd2183124291e9dd2